### PR TITLE
Elmer edits

### PR DESCRIPTION
**Pull request by elmerurbano**
eightball.js
-organized replies
-evened out yes/no/try again replies
-added sass

reply.js
-added a few more replies
-merged dickbutt command into reply.js

### CODE REVIEW

File: src/commands/reply.js [hunk 1]
```diff
@@ -3,11 +3,17 @@ const log = debug('Reply');
...
 responses.set('bot respond', "I'm a pretty stupid bot.");
-responses.set('bot be nice', 'sorry :(');
+responses.set('bot be nice', 'sorry :c');
 responses.set('gj bot', 'thx');
 responses.set('thx bot', 'np');
 responses.set('bot pls', '( ¬‿¬)');
+responses.set('not now bot', 'aww ok :c');
+responses.set('go away bot', 'I just wanted to be friends');
+responses.set('(╯°□°)╯︵ ┻━┻', '┬─┬﻿ ノ( ゜-゜ノ)');
+responses.set( '┬─┬﻿ ノ( ゜-゜ノ)', 'THAT\'S MY JOB');
```
Comment (IanMitchell) [inline src/commands/reply.js]: Use "string's" instead of 'string\'s'

File: src/commands/dickbutt.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const debug = require('debug');
+const log = debug('Dickbutt');
+
+
+const dickbutt_img = 'https://i.imgur.com/etjgJ2D.jpg';
```
Comment (IanMitchell) [inline src/commands/dickbutt.js]: Let's move this to reply

File: src/commands/reply.js [hunk 3]
```diff
@@ -1,13 +1,37 @@
...
+  log(greeting);
+  return greeting;	
+}
+
 const responses = new Map();
 responses.set('ping', 'pong');
+responses.set('sing', 'song');
+//this creates the response at startup time
+//want to dynamically create response
+responses.set('hi bot', randomGreeting());
```
Comment (IanMitchell) [inline src/commands/reply.js]: Does this work, or does it set it to only one greeting?

Comment (elmerurbano) [inline src/commands/reply.js]: It only sets to one greeting when the bot starts up. Don't merge this part in yet. It'll be obsolete when we reimplement reply.js anyway.

**Comment by IanMitchell** [comment]
New reply system is in!

**Pull request closed without merging**